### PR TITLE
Added datatypes sanity test for MySQL Plugin

### PR DESCRIPTION
**Pull request by Aryan-Verma**
Added MySQL Plugin Sanity Test in E2E test for querying all supported datatypes.

Also fixed issues related to unsigned int, unsigned mediumint, unsigned bigint where these are detected as wrong datatypes by MySQL Plugin: [PLUGIN-1505](https://cdap.atlassian.net/browse/PLUGIN-1505), [PLUGIN-1506](https://cdap.atlassian.net/browse/PLUGIN-1506), [PLUGIN-1507](https://cdap.atlassian.net/browse/PLUGIN-1507).

[PLUGIN-1505]: https://cdap.atlassian.net/browse/PLUGIN-1505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PLUGIN-1506]: https://cdap.atlassian.net/browse/PLUGIN-1506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PLUGIN-1507]: https://cdap.atlassian.net/browse/PLUGIN-1507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java [hunk 1]
```diff
@@ -49,6 +49,20 @@ public static int countRecord(String table) throws SQLException, ClassNotFoundEx
     }
   }
 
+  public static int validateRecordValues(String sourceTable, String targetTable)
```
Comment (MrRahulSharma) [inline mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java]: Rename to getRecordDifference

File: mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java [hunk 2]
```diff
@@ -80,6 +94,27 @@ public static void createTargetTable(String targetTable) throws SQLException, Cl
     }
   }
 
+  public static void createSourceDatatypesTable(String sourceTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getMysqlConnection(); Statement statement = connect.createStatement()) {
```
Comment (MrRahulSharma) [inline mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java]: Cosmetic: Post the ';' char use new line.

File: pom.xml [hunk 3]
```diff
@@ -701,7 +701,7 @@
           <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>27.0.1-jre</version>
```
Comment (MrRahulSharma) [inline pom.xml]: This change should be taken separately.

File: mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java (hunk 1 above)
Comment (Aryan-Verma) [reply mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java]: Done

File: mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java (hunk 2 above)
Comment (Aryan-Verma) [reply mysql-plugin/src/e2e-test/java/io/cdap/plugin/MysqlClient.java]: Done

File: pom.xml (hunk 3 above)
Comment (Aryan-Verma) [reply pom.xml]: This is now merged to the develop branch by Vipin

**Review by MrRahulSharma — APPROVED** [review]

**Pull request merged**